### ISSUE TITLE
Streamline service filters with help of Jack and Jenna from Reeb.

### ISSUE THREAD

**Issue by vijayyadav06** [SCODEMeetup/mofb-api#42]
Streamline service filters with help of Jack and Jenna from Reeb.

**Comment by mycarrysun** [comment]
I'd like to make some updates to the UI here.

I think it would make more sense to have a blank welcome screen with buttons for the 3 main categories first (Food, Housing, Transportation), and then show about 5-10 buttons (maybe only 2-3) for a smaller subset of child categories that would then show the locations on the map.

The average person walking by the REEB center is not going to want to read through all the different categories to find what they're looking for.

After taking a look at the code and digging into it I see that we can implement this on the frontend by conditionally showing the map if there are child categories chosen. If there are no categories chosen (i.e. app was just opened), we show 3 buttons that will set a category in the redux state to the parent category ID. Then another set of buttons to choose a subcategory/subcategory group which will set the redux category IDs and load the map.

Thoughts?

I am going to begin implementing this in a PR but if anyone has suggestions to how this may be implemented easier, let me know.

**Comment by vijayyadav06** [comment]
i agree with the idea and your approach for implementing it. @melissa3000  / @mihiramin89 please jump in as well with your comments.

**Comment by mycarrysun** [comment]
Alright so digging into this further I think the main thing we need to fix here is creating a sort so that the most popular items show at the top of the list.

With the new UI I am working on this will work perfectly as there is a "Show More" button to show all possible sub categories. 

Down the road we should probably group some of the categories together so that multiple Taxonomy IDs are requested by choosing a single subcategory but that could be another issue/PR.

**Comment by mycarrysun** [comment]
Seems to be the best solution at the moment: sort the services by their `level` property. It shows some of the more frequently accessed/useful services first.
